examples/complete/variables.tf

```diff
@@ -1,6 +1,7 @@
 variable "region" {
   type        = string
-  description = "AWS region"
+  default     = ""
```

Comment:
This isnt right.

Please see the s3 bucket

https://github.com/cloudposse/terraform-aws-s3-bucket/blob/caf2af9a03f947ae51348e675146f6475ca228a3/examples/complete/main.tf#L1-L9

The awsutils provider needs to be set with a region in examples/complete/main.tf just like above

And only the examples/complete/versions.tf file needs to be updated with this

https://github.com/cloudposse/terraform-aws-s3-bucket/blob/caf2af9a03f947ae51348e675146f6475ca228a3/examples/complete/versions.tf#L13

Reply:
@nitrocode the var.region on the provider is already set as you can see **[here](https://github.com/denissimonovski/terraform-aws-codebuild/blob/5e476c1beb3e58eb1a573c9bf338a5e47951f8a6/examples/complete/main.tf#L2)** and the region is in the examples `.tfvars`. And the awsutils provider is not required in this module at the moment. If you say I should put it in the example, shouldn't I put it in the module as well? Given that the example only deploys this module and nothing else.

Reply:
```hcl
# this is not aws provider, this is the needed provider awsutils
provider "awsutils" {
  region = var.region
}
```

Reply:
By using the latest s3 bucket module the awsutils provider is needed